Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3
+
+ARG PATH_FOR_INITIAL_DATA="restaurants.json"
+ENV PATH_FOR_INITIAL_DATA="${PATH_FOR_INITIAL_DATA}"
+# set a directory for the app
+WORKDIR /app
+
+# copy all the files to the container
+COPY . /app
+
+# install dependencies
+RUN pip install --no-cache-dir -r requirements.txt
+
+# tell the port number the container should expose
+EXPOSE 5000
+
+# run the command
+CMD ["python", "./app.py"]
```

Comment:
`CMD` is used to specify a default command to execute when contain is running. this is fine for  some cases (e.g. for `linux` image to run `bash` terminal by default), but in general `ENTRYPOINT` is preferred:
https://goinbigdata.com/docker-run-vs-cmd-vs-entrypoint/
"The bottom line"

Reply:
fixed